main.py

```diff
@@ -280,52 +272,64 @@ async def on_message(self, message):
         # Debug xpGain
         if debugMode == True: logger.debug(f"xpGain | {xpGain}")
 
-        # Ref to firebase members collection and member doc
-        members_col = db.collection("members")
-        member_query = db.collection("members").where(filter=FieldFilter("member_id", "==", message.author.id))
-        
-        if len(member_query.get()) == 1:
-            # Get member doc from query
-            member_doc = member_query.get()[0]
+        # Mysql query
+        try:
+            cursor.execute(f"SELECT * FROM members WHERE id = {message.author.id}")
```

Comment:
Generally in Discord environment this SQL won't be a problem, but in imaginary world, what would happen if `author` could change its id to `0 OR 1=1`? SQLAlchemy will solve this problems